app/Http/Controllers/QuizAPIController.php

```diff
@@ -23,4 +24,25 @@ public function displayAllQuizzes():JsonResponse
             "data" => $quizzes
         ]);
     }
+
+    public function addNewQuiz(Request $request): JsonResponse
+    {
+        $request->validate([
+            'name' => 'required|string|min:1|max:255',
+            'description' => 'required|string|min:1|max:1000'
+        ]);
+
+        try {
+            $quiz = new Quiz();
+            $quiz->name = $request->name;
+            $quiz->description = $request->description;
+            $quiz->save();
```

Comment:
What happens if the database exists and can connect but there is some internal problem with the database or the data that prevents saving this request?

Example so that you can test easily: Your validation allows a wrong or too-long data type through.

Your validation and tests are supposed to protect you from this specific example, but there are other cases where the database can be up but 'something bad' still happens and it rejects the request, so you should handle this.